packages/design-system/src/styles/components/_UsaBanner.scss

```diff
@@ -107,6 +109,7 @@
 
   @media (min-width: $media-width-md) {
     display: inline-block;
+    grid-area: action;
```

Comment:
because i added the "spacer" cell name above, not defining where this action text lives at md breakpoints caused it to squish inside the first available cell it could find ("spacer") and that's not what we want.